### PR TITLE
Remove custom js rules

### PR DESCRIPTION
**Pull request by hhugo**

### CODE REVIEW

**Comment by hhugo** [comment]
It currently only works with the `release` profile: `dune build @all --profile release`
It fails with the dev (default) profile because the `--file` flag is used twice, making `examples.py` embedded twice and triggering an error at runtime

**Comment by LaurentMazare** [comment]
Thanks!

File: examples/index.html [hunk 1]
```diff
@@ -174,7 +174,7 @@
         fileref.setAttribute("src", prefix+(version==""?"":(version+"/"))+url);
         document.getElementsByTagName("head")[0].appendChild(fileref);
       }
-      load_script("toplevel.js");
+      load_script("toplevel.bc.js");
```
Comment (hhugo) [inline examples/index.html:177]: this should be toplevel_js.bc.js

Comment (LaurentMazare) [reply examples/index.html:177]: Yup just fixed it.

**Pull request merged**